SocialFilter/sections/css_extended.txt

```diff
@@ -2,9 +2,9 @@
 ! Extended CSS rules
 !
 singleclickapps.com#?#.nav-list > li.nav-header:contains(/\sUs on\s/)
-geo.de##.social-bar > dd:has(> a.fb-button)
-geo.de##.social-bar > dd:has(> a.twitter-button)
-geo.de##.sidebar > section.sidebar-teasergroup:has(> iframe[src^="https://www.facebook.com/"])
+geo.de#?#.social-bar > dd:has(> a.fb-button)
+geo.de#?#.social-bar > dd:has(> a.twitter-button)
+geo.de#?#.sidebar > section.sidebar-teasergroup:has(> iframe[src^="https://www.facebook.com/"])
```

Comment:
Please see previous comment about facebook text since that iframe is not always shown in code.